### ISSUE TITLE
Hello, I have a question regarding obfuscated datasets.

### ISSUE THREAD

**Issue by kimbjunn** [OSUSecLab/SymGen#3]
Hi, thank you so much for sharing your amazing research.

In the process of reproducing the experiment in the paper, it is mentioned that the obfuscated dataset from SymLM was utilized, but I couldn't find it in the SymLM repository. Do you have an accessible link?

Also, in the experiment, is it true that the obfuscated binary was utilized only as a test set for the existing baseline experimental model trained with non-obfuscated binaries?

Thank you.

**Comment by Shinaaaaaa** [comment]
Hi @kimbjunn,

Thank you for your interest and for attempting to reproduce our results!

I’ve uploaded the obfuscated dataset here: https://zenodo.org/records/15598737. And this obfuscated dataset is only used as a test set and was not used during any stage of training.

**Comment by kimbjunn** [comment]
Thank you very much for your quick and kind reply.
Is there any binary level dataset that does not exist?

**Comment by Shinaaaaaa** [comment]
Sorry, I may not have fully understood your question. If you’re referring to the dataset used in the paper, you can find it at the Zenodo link, and the malware dataset comes from this repository: https://github.com/vxunderground/MalwareSourceCode/tree/main/Linux

**Comment by kimbjunn** [comment]
Oh, I was talking about obfuscated datasets. The obfuscated dataset you shared is identified as a set of preprocessed .json files. Could you please share the obfuscated binary level dataset before preprocessing?

**Comment by Shinaaaaaa** [comment]
Hi @kimbjunn, we have uploaded it to Zenodo.

**Comment by kimbjunn** [comment]
Thank you @Shinaaaaaa.
And the repository doesn't reveal the trained model, but it seems like there is a source code and explanation of the process of reproducing the fine-tuning. Am I understanding it correctly?

**Comment by Shinaaaaaa** [comment]
Yes, you can fine-tune your own model by following the [Fine-tune and Predict](https://github.com/OSUSecLab/SymGen?tab=readme-ov-file#fine-tune-and-predict) section in the README.md
